src/MDMPerforming.h

```diff
@@ -114,3 +114,31 @@ NS_SWIFT_NAME(ComposablePerforming)
     NS_SWIFT_NAME(set(transactionEmitter:));
 
 @end
+
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+@optional
```

Comment:
Making both methods required on the protocol will require that an object conforming to the protocol implements both methods.